src/memory-analyzer/gdb_api.cpp

```diff
@@ -0,0 +1,474 @@
+/*******************************************************************\
+
+Module: GDB Machine Interface API
+
+Author: Malte Mues <mail.mues@gmail.com>
+        Daniel Poetzl
+
+\*******************************************************************/
+
+/// \file
+/// Low-level interface to gdb
+
+#ifdef __linux__
```

Comment:
Which part of this code is really Linux-specific? If any at all? Even when we might never be able to run a `gdb` binary on Visual Studio, it's good to know that at least the code compiles and can be re-used/adapted.

Reply:
Yes, changed to allow various Unices. We need the `fork()` call from `unistd.h`.